addons/skin.estuary/xml/Variables.xml

```diff
@@ -258,15 +258,15 @@
 	</variable>
 	<variable name="VideoInfoMainLabelVar">
 		<value condition="String.IsEqual(ListItem.DBType,set)">$INFO[ListItem.Title]</value>
-		<value condition="!String.IsEmpty(ListItem.Title)">$INFO[ListItem.Title]$INFO[ListItem.Year, ([COLOR grey],[/COLOR])]</value>
+		<value condition="!String.IsEmpty(ListItem.TVShowTitle)">$INFO[ListItem.TVShowTitle]$INFO[ListItem.Year, ([COLOR grey],[/COLOR])]</value>
```

Comment:
ListItem.Title is used for other types of content, so please do not remove that line.
just place the ListItem.TVShowTitle one above the ListItem.Title one and you should be fine.

Reply:
Ok, will do